addons/web/static/src/core/install_prompt/install_prompt.xml

```diff
@@ -8,27 +9,32 @@
                     <div t-on-click="onClose" type="button" class="btn-close" aria-label="Close"/>
                 </div>
             </t>
-            <p>Install Odoo on your device to access it easily. Here are the steps to follow:</p>
-            <t t-if="isMobileSafari">
-                <ul class="list-group list-group-numbered">
-                    <li>
-                        Tap on the share icon
-                    </li>
-                    <li>
-                        Select "Add to home screen"
-                    </li>
-                </ul>
-            </t>
-            <t t-else="">
-                <ul class="list-group list-group-numbered">
-                    <li>
-                        Open "File" menu from your browser
-                    </li>
-                    <li>
-                        Select "Add to dock"
-                    </li>
-                </ul>
-            </t>
+            <p>Install the app on your device to access it easily. Here are the steps to follow:</p>
+            <t t-call="web.InstallStepsSafari"/>
         </Dialog>
     </t>
+
+    <t t-name="web.InstallStepsSafari">
```

Comment:
if safari is the culprit here; why not shame them in the code?
![image](https://github.com/odoo/odoo/assets/9530767/7728a955-6a62-4ace-96f2-4bcdf300a324)

Reply:
Yes, I prefer to keep naming Safari, because the steps listed in this template only applies to Safari mobile and desktop (tap share -> add to home screen _or_ click on file -> add to dock)

Reply:
I'm asking to change the name because this part of the template is executed when not in Safari: https://github.com/odoo/odoo/pull/152699/files#diff-928955c0ad1ea6f8d4615719e02ab3f2c8a0dd85a99eccd0506052bca5cd1be8R26-R34